### PR TITLE
Milestone 1: Create Database, Add Data and Query DB

### PR DESCRIPTION
**Pull request by AbayomiOlaoye**
### Highlight

In this pull request, I worked on a database using postgreSQL to:
- [x] Create vet_clinic's database
- [x] Add a table data for animals' information management
- [x] Include several queries that used to classify data into several useful divisions: 
- Find all animals whose name ends in "mon"
like_ends_mon

- List the name of all animals born between 2016 and 2019.
between2016_2019

- List the name of all animals that are neutered and have less than 3 escape attempts.
escape_neutered

- List the date of birth of all animals named either "Agumon" or "Pikachu".
DOB

- List name and escape attempts of animals that weigh more than 10.5kg
name_attempts

- Find all animals that are neutered.
neutered

- Find all animals not named Gabumon.
not_gabumon

- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
greater_equal

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE date_of_birth  BETWEEN '2016-01-01' and '2019-12-31';
+SELECT * from ANIMALS WHERE neutered=true and escape_attempts<3;
+select date_of_birth from animals where name='Agumon' or name='Pikachu';
+select (name, escape_attempts) from animals where weight_kg>10.5;
+select * from animals where neutered=true;
+select * from animals WHERE NOT name='Gabumon';
+select * from animals where weight_kg>=10.4 or weight_kg>=17.3;
```
Comment (cynthiainga) [inline queries.sql]: - [ ] Good job so far but here the query should find all animals with a weight between 10.4kg and 17.3kg, so you should use `AND` and not `OR` also the second weight should be less or equal to `<=`. eg: `select * from animals where weight_kg >= '10.4' and weight_kg <= '17.3';`. Please kindly fix that 🙃

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE date_of_birth  BETWEEN '2016-01-01' and '2019-12-31';
```
Comment (cynthiainga) [inline queries.sql]: - [OPTIONAL] In this query you don't need to select all you just need to select the name from animals `SELECT name from animals WHERE date_of_birth  BETWEEN '2016-01-01' and '2019-12-31';` 😉 😉

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT * from ANIMALS WHERE neutered=true and escape_attempts<3;
```
Comment (cynthiainga) [inline queries.sql]: - [OPTIONAL] In this query you don't need to select all as well, you just need to select the name from animals `SELECT name from animals WHERE neutered = true and escape_attempts < 3;`

File: README.md [hunk 4]
```diff
@@ -244,44 +163,25 @@ Feel free to check the [issues page](../../issues/).
...
-
-  - [Answer_2]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (cynthiainga) [inline README.md]: - [OPTIONAL] Please kindly add the correct link file to your project because the link that you provided in your README file is broken 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @AbayomiOlaoye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (AbayomiOlaoye) [reply README.md]: I'll fix all those, thanks! That was an omission.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @AbayomiOlaoye👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**